libs/remix-ui/workspace/src/lib/remix-ui-workspace.tsx

```diff
@@ -156,6 +159,14 @@ export function Workspace () {
     }
   }
 
+  const onFinishDownloadWorkspace = async () => {
+    try {
+      await global.dispatchHandleDownloadWorkspace()
+    } catch (e) {
+      console.error(e)
```

Comment:
do we need a proper error report in UI here?

Reply:
ok, will show some error on UI too

Reply:
done